language/stdlib/modules/VASP.move

```diff
@@ -117,6 +117,7 @@ module VASP {
     /// Rotate the compliance public key for `parent_vasp` to `new_key`
     public fun rotate_compliance_public_key(parent_vasp: &mut ParentVASP, new_key: vector<u8>) {
         Transaction::assert(Vector::length(&new_key) == 32, 7004);
```

Comment:
You can drop this check because `validate_pubkey` fails if  the length is invalid.